stock_analytic/tests/test_stock_picking.py

```diff
@@ -85,12 +106,12 @@ def _create_picking(
         return picking
 
     def _confirm_picking_no_error(self, picking):
-        picking.action_confirm()
+        picking.action_assign()
         self.assertEqual(picking.state, 'confirmed')
 
     def _force_assign_no_error(self, picking):
-        picking.force_assign()
```

Comment:
@aheficent As reading Odoo message : https://github.com/odoo/odoo/blob/12.0/addons/stock/models/stock_picking.py#L705

IMHO You need to adapt quantities to simulate the former 'force'. You change the test behaviour here.